magellan-library/src/main/java/com/wealthfront/magellan/transitions/Transition.kt

```diff
@@ -0,0 +1,38 @@
+package com.wealthfront.magellan.transitions
+
+import android.view.View
+import com.wealthfront.magellan.Direction
+
+/**
+ * Define a transition (animation) between two screens. By default, transitions are implemented by
+ * [DefaultTransition]. You can either set a different default one when building your Navigator, using
+ * [Navigator.Builder.transition],
+ * or override the next transition by using
+ * [Navigator.overrideTransition].
+ *
+ *
+ * You can find more Transitions implemented in [com.wealthfront.magellan.transitions].
+ */
+interface Transition {
```

Comment:
Since we're making a breaking change to this interface anyway, how do you feel about renaming this to a `StepTransition` or `MagellanTransition` so it doesn't conflict with android's `Transition` class?

Reply:
Done